Makefile.include

```diff
@@ -2,7 +2,7 @@
 # Local customization
 ##################################################
 
-SELF_DIR=$(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST)))))
+SELF_DIR:=$(realpath $(dir $(realpath $(lastword $(MAKEFILE_LIST)))))
```

Comment:
What's the difference in Makefile semantics?

Reply:
Shouldn't it be `?=` instead ?

Reply:
In Make world, it is a "simply expanded variable". `:=` means the value of `SELF_DIR` is computed (Make says "expanded") once and for all at definition time. Equivalent to say I'm passing the value of the expression as the variable value, and not the expression itself. 
So `SELF_DIR` will stay in the following what it was in this first line.
`=` are recursively expanded. The expression is evaluated when the variable is referenced… so if someone includes a makefile from another directory, then it becomes the last of the `MAKEFILE_LIST` and the `SELF_DIR` is now its directory instead of Mlang's.
I ran into this because my DGFiP config file includes directly one of the old makefile from the existing system, in order to load the C sources file lists corresponding to the different target without duplicating them.

Reply:
Wouahou this should be in a comment, otherwise somebody is going to remove the `:` in the future with good intentions but catastrophic consequences :)

Reply:
Example: let's say I'm going `rm -rf` a subdir of Mlang in a clean command and I use `SELF_DIR` as a base of my path. Suddenly I just wiped another totally unrelated and badly-versionned subdir of the server. This is based on real world stunt, do not try to reproduce at home.

Reply:
You can see I used it also in commit [d63a2b2](https://github.com/MLanguage/mlang/pull/216/commits/d63a2b2988503672440229185ff1ebe98571c7fe), in order to ensure shell call is only done once. Otherwise, if you define function in makefiles to call in variable assignments, the function is called every time the variable is used. Not so visible with a short `find` on 30 files, but a bit painful with SVN update or log crawl on the whole repository.